src/libmusig2.c

```diff
@@ -281,6 +281,12 @@ int musig2_sign(musig2_context_sig *mcs, const unsigned char *msg, int msg_len,
     secp256k1_xonly_pubkey xonly_aggr_pk;   // x_only aggregated public key
     musig2_param param;  // Parameters used to generate partial signature
 
+    int j;
+    int index = V * mcs->mc->state;
+    for (j = 0; j < V; j++)
+        if (mcs->comm_list[index + j] == NULL)
+            return -1;
+
```

Comment:
Given that we now have more than one error type, we need to include the description in the documentation of the function.